apps/machine/zynq7/platform_info.h

```diff
@@ -11,11 +11,17 @@
 #include <openamp/remoteproc.h>
 #include <openamp/virtio.h>
 #include <openamp/rpmsg.h>
+#include <xparameters.h>
 
 #if defined __cplusplus
 extern "C" {
 #endif
 
+#define RING_TX                     FW_RSC_U32_ADDR_ANY
+#define RING_RX                     FW_RSC_U32_ADDR_ANY
+#define NUM_VRINGS                  0x02
+#define VRING_ALIGN                 0x1000
+
```

Comment:
In apps/machine/zynqmp_r5/platform_info.h you also move the  VRING_SIZE, What about doing same here?